src/ui/popup/App.tsx

```diff
@@ -421,6 +305,77 @@ class App extends React.Component<PopupAppComponentProps, InitialState> {
             text={browser.i18n.getMessage('preferencesText')}
           />
         </div>
+        <div
```

Comment:
It's a start, but what I meant was this whole entire UI portion (308-378) should be separated into its own file to make the main file smaller.

Reply:
Resovled through d02a4b6